### DISCUSSION TITLE
Questions about the properties of the time bar

### DISCUSSION THREAD

**Discussion by GitandGeek** [frappe/insights#340] [Q&A]
I'm currently building an interactive chart based on this repo. I have two questions: 1. Can the left edge of the time bar be dragged to adjust the start time? 2. Is it possible to change the colorless completed part of the time bar to another color, say, green?

**Comment by nextchamp-saqib** [comment]
> 1. Can the left edge of the time bar be dragged to adjust the start time?

No, this kind of interactivity is not supported by `echarts`, as far as I know.

> 2. Is it possible to change the colorless completed part of the time bar to another color, say, green?

This should be possible; you may need to explore the `echart` API options to achieve this.
